reference/forms/types/language.rst

```diff
@@ -74,6 +75,19 @@ If this option is ``true``, the choice values use the `ISO 639-2 alpha-3`_
 three-letter codes (e.g. French = ``fra``) instead of the default
 `ISO 639-1 alpha-2`_ two-letter codes (e.g. French = ``fr``).
 
+choice_self_translation
```

Comment:
By only looking at the option name I would not really know what this is supposed to do. Can we think of something more self-explanatory?

Reply:
I agree with you. I've updated the description. Is it more clear now? Thanks.

Reply:
I was rather think about renaming the option. But sadly I cannot come up with a better name. :(

Reply:
But we can discuss that separately.